chromadb/api/models/Collection.py

```diff
@@ -29,6 +29,9 @@
     from chromadb.api import API
 
 
+default_embedding_function = None
```

Comment:
Would it be cleaner to push the singleton down to the EF level?

Reply:
No - just for the default we'd change the class so it only has a singleton model, rather than making the singleton live at this level.

Reply:
Oh, make sense, will do.